TimberAPI/AssetLoaderSystem/ResourceAssetPatch/TimberApiResourceAssetLoader.cs

```diff
@@ -47,22 +61,28 @@ public TimberApiResourceAssetLoader(IAssetLoader assetLoader)
             return Resources.LoadAll<T>(path);
         }
 
-        private static void FixMaterialShader(GameObject obj, Shader shader)
+        private static void FixMaterialShader(GameObject obj)
         {
             var meshRenderer = obj.GetComponent<MeshRenderer>();
             if (meshRenderer)
             {
                 foreach (Material mat in meshRenderer.materials)
                 {
-                    mat.shader = shader;
+                    
+                    TimberAPIPlugin.Log.LogFatal(mat.renderQueue);
```

Comment:
Logging used for debugging? Should be removed

Reply:
Ups, yea I had some problems again with versioning, so they were removed but not in this version. (Almost deleted everything again)

Reply:
:D sounds like classic stuff with git